src/steps/fetch-enterprise-accounts/index.ts

```diff
@@ -27,15 +28,7 @@ const step: IntegrationStep = {
       return createIntegrationEntity({
         entityData: {
           source: {
-            id: e.id,
-            name: e.name,
-            createdAt: e.created_at,
-            updatedAt: e.updated_at,
-            trial: e.trial,
-            permissions: e.permissions ? e.permissions.toString() : null, // parse list to string?
-            identityProviderId: e.identity_provider
-              ? e.identity_provider.id
-              : null,
+            ...keysToCamel(e),
```

Comment:
As mentioned in another PR, `source` should be the unmodified source data. `assign` is meant to be used for properties that cannot be derived from the `source` data. For example:

```
createIntegrationEntity({
  entityData: {
    source: e,
    assign: {
      _type: 'heroku_account',
      _class: 'Account',
      createdOn: getTime(e.created_at),
      updatedOn: getTime(e.updated_at),
    },
})
```

`name` is auto-transferred (a test for a converter for the `heroku_account` entities would verify that). We want `createdOn` etc. to be `number` type, to conform to the data model.

Reply:
Makes sense. I originally made this change to auto-import those `createdOn` and `updatedOn` fields but I'll change move towards manually adding properties to `assign`